package.json

```diff
@@ -54,6 +54,7 @@
     "@tweenjs/tween.js": "^18.6.4",
     "earcut": "^2.2.3",
     "js-priority-queue": "^0.1.5",
+    "lodash": "^4.17.21",
```

Comment:
it's necessary?

Reply:
Indeed, it appears that `lodash` is actually not mandatory.